DataLink.tex

```diff
@@ -368,7 +369,7 @@ \subsection{Registering \blinks\ endpoints}
 Operators still wishing to declare \blinks\ endpoints can do this by
 giving a capability with a standardID of
 \begin{verbatim}
-   ivo://ivoa.net/std/DataLink#links-1.0
+   ivo://ivoa.net/std/DataLink#links-1.1
 \end{verbatim}
 Note that this (minor-versioned) URI is applicable to endpoints following any version 1.*
```

Comment:
In view of this change, there needs to be some adjustment to the text in sec 2.2:
> "Note that this (minor-versioned) URI is applicable to endpoints following any version 1.*".

I think just deleting it would be OK, but possibly there should be advice to clients about how to treat this version ID (match against `#links-1.*`).